systemtest/src/test/java/io/strimzi/systemtest/LogSettingST.java

```diff
@@ -479,6 +482,138 @@ void testJSONFormatLogging() {
         assertThat(StUtils.checkLogForJSONFormat(eoPods, "user-operator"), is(true));
     }
 
+    @Test
+    @Order(14)
+    void testDynamicallySetEOloggingLevels() {
```

Comment:
Maybe I missed it. But I'm not sure where this really tests the reloading. But I do not see any test that the `log4j2.properties` was reloaded and the new log level is used. I think that is the mainpoint odf this change so it should be tested.

Reply:
It is testing th e `log4j2.properties` has been changed: https://github.com/strimzi/strimzi-kafka-operator/pull/2867/files/eef5f038315cc85ca5b732297d1a140c8b668d45#diff-406f189535ad433ebe2d832137206aa6R502

You are right we should also test the change really takes an effect. I gues I need help from @Frawless @see-quick there.

Reply:
https://github.com/strimzi/strimzi-kafka-operator/blob/master/systemtest/src/test/java/io/strimzi/systemtest/RollingUpdateST.java#L689   Maybe we can add there a check that logging config file rly changed, but it's not a good idea to see if new debug level is available in logs, because `oc cluster up` is not friendly with that and from time to time just drop the logs

Reply:
I can understand that the reliability might be an issue here. But without that we should maybe consider the usefulness of this test maybe? Right now it really tests that Kube refreshs the config map volume. Does that need to be tested? Is it worth the testing time? If we actually check the logs, I would say yes. Without it? I'm not sure.

Reply:
I think this comment stands ... this test does not test dynamic logging reconfiguration. It just tests how Kuberetes update the mounted config map.

Reply:
I see. IMHO it has to be tested by ST.

Reply:
Why does the update of the ConfigMap which is Kubernetes thing need to be tested by our ST?